maquetado/scss/detail.scss

```diff
@@ -2,30 +2,66 @@
 
 .book-title {
   font-size: 32px;
+  font-family: Roboto;
   font-weight: bold;
   margin-right: 10px;
+  font-size: 32px;
+  line-height: 38px;
 }
 
 .book-titleDescription {
+  font-family: Roboto;
   color: $color-black;
   font-size: 20px;
   font-weight: bold;
   margin-right: 10px;
+  font-size: 20px;
+  line-height: 24px;
 }
 
 .book-description {
+  font-family: Roboto;
   color: $color-grey;
   font-size: 20px;
+  font-size: 20px;
+  line-height: 24px;
 }
 
 .book-genre {
   font-size: 24px;
   font-weight: bold;
+  font-size: 32px;
+  line-height: 38px;
 }
 
 .book-detail {
   display: flex;
   justify-content: flex-start;
   margin-left: 10px;
   margin-top: 20px;
-}
\ No newline at end of file
+}
+
+.book-content {
+  background-color: $color-white;
+  box-shadow: 4px 4px 10px 0 rgba(0, 0, 0, 0.5);
```

Comment:
Can `4px` be changed for `5px`?

Reply:
For this type of colors we usually use 

`$black-05` when you need to add some opacity to it.